### PR TITLE
Improvement/backup routine

### PR DESCRIPTION
**Pull request by jsmatias**
Hotfix: The flag `--users` ensures that the users details are also included in the exported realm file.

### CODE REVIEW

File: scripts/backup.sh
Comment (jsmatias) [inline scripts/backup.sh:1]: The permissions were changed for these files, but not the content.

File: scripts/realm_export.sh [hunk 1]
```diff
@@ -9,7 +9,7 @@ source .env
 DATA_PATH=$(realpath "$DATA_PATH")
 LOCAL_BACKUP_PATH="$DATA_PATH"/keycloak_realm
 
-docker exec -i keycloak /bin/bash -c "/opt/keycloak/bin/kc.sh export --file /tmp/aiod.json --realm aiod"
+docker exec -i keycloak /bin/bash -c "/opt/keycloak/bin/kc.sh export --file /tmp/aiod.json --realm aiod --users realm_file"
```
Comment (PGijsbers) [inline scripts/realm_export.sh:12]: Why did you choose the `realm_file` option? I am not that familiar to keycloak yet, but to me it seems that separate file(s) for users through `same_file` or `different_files` to save users independently from configuration is very useful. You can easily share/back-up configurations without sharing users. You can also rollback one without rolling back the other. For simplicity, `same_file` is probably fine.

c.f. https://www.keycloak.org/server/importExport

**Review by PGijsbers — COMMENTED** [review]
See remark. I don't _need_ a change, but I want motivation for the choice before accepting.

File: scripts/realm_export.sh (hunk 1 above)
Comment (PGijsbers) [reply scripts/realm_export.sh:12]: Created #304 so we can merge this now.

**Review by PGijsbers — APPROVED** [review]
LGTM, created #304 as follow up.

**Pull request merged**